source/administration/migrating.rst

```diff
@@ -64,19 +64,23 @@ Messages with file attachments are imported as a message containing a link to Sl
 
 Bot and Integration messages are imported by default, but if you would like them to display with the appropriate username when imported, you should ensure that `Enable Integrations to Override Usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`_ is set in **System Console > Integrations > Custom Integrations** *before* doing the import.
 
+When Slack messages that are topic change messages, purpose change messages, and channel name change messages are imported, they appear in Mattermost as posts from the System user.
```

Comment:
Can we condense this sentence a bit? Maybe something like "When Slack messages about changes to the topic, purpose, or channel name are imported, they appear in Mattermost as posts from the System user."